core/src/components/range/range.tsx

```diff
@@ -40,6 +40,24 @@ export class Range implements ComponentInterface {
    */
   @Prop() mode!: Mode;
 
+  /**
+   * The neutral point of the range slider.
+   * Default: value is `0` or the `min` when `neutralPoint < min` or `max` when `max < neutralPoint`.
+   */
+  @Prop() neutralPoint = 0;
+  protected neutralPointChanged() {
+    if (!this.noUpdate) {
```

Comment:
I typically prefer to check the positive case when possible. It makes the code a bit more cleaner/readable and avoids extra levels of indentation.

In this case, you'd do `if (this.noUpdate) { return }`

Reply:
you are absolutly right, but this is done everywhere in the component, where `noUpdate` is used, so i wanted to make less changes as possible to the original code, but keep the "form" how the old code is structured.

But i will change it.

Thanks for looking into it :)

Reply:
Fair point! This is something we should probably change elsewhere as well. Thanks for bringing this to our attention 😄

Reply:
should be solved 🙃